models/health_config.go

```diff
@@ -0,0 +1,26 @@
+package models
+
+// Annotationkey is a mnemonic type name for string
+type AnnotationKey string
+
+const (
+	AllHealthAnnotation  AnnotationKey = ".*"
+	RateHealthAnnotation AnnotationKey = "health.kiali.io/rate"
+)
+
+func GetHealthConfigAnnotation() []AnnotationKey {
+	return []AnnotationKey{RateHealthAnnotation}
+}
+
+func GetHealthAnnotation(annotations map[string]string, filters []AnnotationKey) interface{} {
```

Comment:
Perhaps "GetHealthAnnotation" should return a map[string]string instead of "interface{}".

Also, below there are multiple lines doing the conversion on the call:

```
GetHealthAnnotation(service.Annotations, GetHealthConfigAnnotation()).(map[string]string)
```

I guess if not found or if there is a type class it can return an error or a nil.

Reply:
I remember that i had problems in graph/telemetry/istio/appender/health_config.go when I return that to the metadata let me check

Reply:
mmm I did the commit with changes I am going to tests all again because I remember something about this